Extensions/retags.js

```diff
@@ -102,13 +106,12 @@ XKit.extensions.retags = {
 				   $target = $element;
 				   url = $target.find('.action').data('post-url');
 			// Activity
-			} else if ($element.hasClass('ui_note')) {
-				if ($element.find('.part_response').length) {
-					$element.addClass('is_response');
-				}
+            } else if ($element.hasClass('activity-notification')) {
 				retagClass = 'is_retags';
 				$target = $element;
-				url = $target.find('.part_glass').attr('href');
+                var glass = $($element).find('.activity-notification__glass');
+                host = glass.data('peepr').tumblelog + '.tumblr.com';
```

Comment:
I think just making host the tumblog name *should* work, because that's how it gets passed into the API